javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/OpenTelemetryInstaller.java

```diff
@@ -46,8 +46,6 @@ public static synchronized void installAgentTracer(Config config) {
       if (config.getBooleanProperty(JAVAAGENT_NOOP_CONFIG, false)) {
         GlobalOpenTelemetry.set(NoopOpenTelemetry.getInstance());
```

Comment:
If the noop context implementation is no longer auto-discoverable, should we set it explicitly here?

I wonder if we're just trading one `System.setProperty()` for another, in other words...